packages/web-app-files/src/mixins/fileActions.js

```diff
@@ -245,20 +247,23 @@ export default {
           class: `oc-files-actions-${app.name}-trigger`,
           isEnabled: () => true,
           canBeDefault: defaultApplication === app.name,
-          handler: () => this.$_fileActions_openLink(app.name, fileId),
+          handler: () => this.$_fileActions_openLink(app.name, webDavPath, app.name),
```

Comment:
Why do you pass in `app.name` a second time?

Reply:
🤦